src/containers.rs

```diff
@@ -0,0 +1,305 @@
+        match message {
+            Self::Input::Docker(switched_on) => {
+                tracing::info!(
+                    "{}",
+                    if switched_on {
+                        "Enabling Docker"
+                    } else {
+                        "Disabling Docker"
+                    }
+                );
+
+                self.enable_docker = switched_on;
+
+                widgets
+                    .distrobox
+                    .set_sensitive(self.enable_docker || self.enable_podman);
+                if !(self.enable_docker || self.enable_podman) {
+                    widgets.distrobox_switch.set_active(false);
+                }
+            },
+            Self::Input::Podman(switched_on) => {
+                tracing::info!(
+                    "{}",
+                    if switched_on {
+                        "Enabling Podman"
+                    } else {
+                        "Disabling Podman"
+                    }
+                );
+
+                self.enable_podman = switched_on;
+
+                widgets
+                    .distrobox
+                    .set_sensitive(self.enable_docker || self.enable_podman);
+                if !(self.enable_docker || self.enable_podman) {
+                    widgets.distrobox_switch.set_active(false);
+                }
+            },
+            Self::Input::Distrobox(switched_on) => {
+                tracing::info!(
+                    "{}",
+                    if switched_on {
+                        "Enabling Distrobox"
+                    } else {
+                        "Disabling Distrobox"
+                    }
+                );
+
+                self.enable_distrobox = switched_on;
+            },
+            Self::Input::Apptainer(switched_on) => {
+                tracing::info!(
+                    "{}",
+                    if switched_on {
+                        "Enabling Apptainer"
+                    } else {
+                        "Disabling Apptainer"
+                    }
+                );
+
+                self.enable_distrobox = switched_on;
+            },
+            Self::Input::QEMU(switched_on) => {
+                tracing::info!(
+                    "{}",
+                    if switched_on {
+                        "Enabling QEMU"
+                    } else {
+                        "Disabling QEMU"
+                    }
+                );
+
+                self.enable_qemu = switched_on;
+            },
+            Self::Input::Virtualbox(switched_on) => {
+                tracing::info!(
+                    "{}",
+                    if switched_on {
+                        "Enabling Virtualbox"
+                    } else {
+                        "Disabling Virtualbox"
+                    }
+                );
+
+                self.enable_virtualbox = switched_on;
+            },
+        }
+
+        let mut commands: Vec<&str> = Vec::new();
+
+        if self.enable_podman {
```

Comment:
I think this is supposed to be `enable_docker`, but this is also the wrong docker to use. `docker.io` bad and deprecated. Should be using pacstall `docker-bin` or `docker-desktop-deb` (which I need to make pacscript for)

Reply:
also `docker-buildx-plugin-bin` and `docker-compose-plugin-bin`

`docker-desktop` has no `arm64` version either

Reply:
Make the `docker-desktop-deb` pacscript